### ISSUE TITLE
add 'df' as alias for `pd.DataFrame`  in `options`

### ISSUE THREAD

**Issue by fzumstein** [xlwings/xlwings#1533]
Reading can be a bit verbose for something so basic. Instead of:

```python
df = mysheet['A1'].expand().options(pd.DataFrame).value
```
or 

```python
df = mysheet['A1'].options(pd.DataFrame, expand='table').value
```

You could do:

```python
df = mysheet['A1'].options('df', expand='table').value
```

Maybe it should even default to expand='table' which would simplify things to:

```python
df = mysheet['A1'].options('df').value
``` 

The behavior of `pd.DataFrame` could be left as-is to be backward compatible.

Opinions?

**Comment by ericbmoreira** [comment]
I like the idea but I had a bunch of issues with expand='table', so I'm not sure if I like that as a default. Maybe I am an outlier.

Thanks.

**Comment by fzumstein** [comment]
thanks @ericbmoreira, have you ever tried to use `current_region` instead of the expand option to see if that would solve your issues? It's not exactly the same though, see: See: https://docs.xlwings.org/en/stable/api.html#xlwings.Range.current_region

```python
mysheet['A1'].current_region.options(pd.DataFrame).value
```

**Comment by ericbmoreira** [comment]
Thank you for that. I have a note in my code about the closed bug #1164. I believe my issues was having UDF's referencing other UDF's that used the expand='table'. If I ever revisit my code I might try it again or this method.

**Comment by fzumstein** [comment]
Ah ok in the context of UDFs that makes sense and you can ignore the `current_region` comment.
